src/metrics/metrics.go

```diff
@@ -143,12 +145,18 @@ func populateCustomMetrics(instanceEntity *integration.Entity, connection *conne
 		prefix = "USE " + query.Database + "; "
 	}
 
+	log.Debug("Running custom query: %+v", query)
+
 	rows, err := connection.Queryx(prefix + query.Query)
 	if err != nil {
 		log.Error("Could not execute custom query: %s", err)
 		return
 	}
 
+	if !rows.NextResultSet() {
+		log.Error("No result set found for custom query: %+v", query)
```

Comment:
I'm not sure if this should be just a warning (there are no results but the integration works, right?)

Reply:
You're right, much better. Updated here a08e5c7a98bb8fc615e0607553659aca7a2e2b67.